app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
```

Comment:
@dcox Consider adding a `before_action :authenticate_user!` to `ApplicationController` to enforce authentication by default for all controller actions.